src/test/scala/uk/gov/nationalarchives/api/update/utils/ExternalServicesTest.scala

```diff
@@ -98,22 +98,11 @@ class ExternalServicesTest extends AnyFlatSpec with BeforeAndAfterEach with Befo
     wiremockGraphqlServer.stop()
     wiremockAuthServer.stop()
     wiremockKmsEndpoint.stop()
-    api.shutdown()
   }
 
   override def afterEach(): Unit = {
     wiremockAuthServer.resetAll()
     wiremockGraphqlServer.resetAll()
     wiremockKmsEndpoint.resetAll()
-    client.receiveMessage(
```

Comment:
How does the queue get reset at the end of each test? Won't the messages be preserved between tests?

In tdr-download-files, I copied Sam's approach of creating the queue before each test and deleting it after: https://github.com/nationalarchives/tdr-download-files/pull/16/files#diff-406bafae2f04526e067584d16326c8f7c15b2a76a8741cacaf57a4cba1eb521dL82

Reply:
Good point.